cypress/e2e/components/footer.cy.js

```diff
@@ -1,18 +1,17 @@
 import { componentSelectors, modSelectors } from '../constants/selectors';
 import { routes } from '../constants/url';
 
-describe('check footer and content', () => {
+describe('check footer section and banner', () => {
     beforeEach(() => {
         cy.visit(routes.home);
     });
-    context('check contentcards and contents', () => {
+    context('check footer logo', () => {
         it('check svg', () => {
             cy.get(componentSelectors.bottomFooter).within(() => {
-                cy.get('svg.bottomFooter__logo').should('be.visible');
+                cy.get('svg').should('be.visible');
             });
         });
         it('check telephone number', () => {
```

Comment:
ich würde in diesem Testfall prüfen ob der Link `tel:` beinhaltet mit `.includes()`

Reply:
tel: wird nun geprüft und nicht die ganze exakte telefonnummer